### PR TITLE
Update 5-5-0.adoc

### PR DESCRIPTION
**Pull request by jgardiner68**
Suggested changes

### CODE REVIEW

File: docs/modules/release-notes/pages/5-5-0.adoc [hunk 1]
```diff
@@ -1,12 +1,12 @@
 = 5.5.0 Release Notes
 :description:
-These release notes list any new features, enhancements, and fixes that were made between version 5.4.1 and 5.5.0 of Hazelcast Management Center (MC).
+These release notes list any new features, enhancements, and fixes implemented between Hazelcast Management Center (MC) versions 5.4.1 and 5.5.0.
```
Comment (promanenko) [inline docs/modules/release-notes/pages/5-5-0.adoc:3]: this is a standard description we have in all release notes. If we really need to change it then let's change it for all RNs altogether in a separate PR

File: docs/modules/release-notes/pages/5-5-0.adoc [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
 {description}
 
-NOTE: This only applies if you are upgrading your MC from a version before 5.4.0: MC 5.5.0 uses H2 2.x as its internal database instead of H2 1.x used by the previous MC releases.
-H2 1.x and 2.x are not compatible so a data migration step is required.
+NOTE: This only applies if you upgrade your MC from a version before MC 5.4.0: MC 5.5.0 uses H2 2.x as its internal database instead of H2 1.x used by the previous MC releases.
```
Comment (promanenko) [inline docs/modules/release-notes/pages/5-5-0.adoc:6]: copied from 5.4.0

File: docs/modules/release-notes/pages/5-5-0.adoc [hunk 3]
```diff
@@ -21,26 +21,26 @@ a custom run script, refer to the documentation for comprehensive instructions o
 
 == Enhancements
 * Improved logging of H2 Database upgrade process. [MC-2668]
-* Improved styling of WAN Replication table. [MC-2711]
+* Improved styling of the WAN Replication table. [MC-2711]
```
Comment (promanenko) [inline docs/modules/release-notes/pages/5-5-0.adoc:24]: Can we have `a WAN Replication table` and `the WAN Replication table` or is it always just `WAN Replication table`? Previously we mostly didn't use articles in such cases. I summon @oliverhowell

File: docs/modules/release-notes/pages/5-5-0.adoc [hunk 4]
```diff
@@ -21,26 +21,26 @@ a custom run script, refer to the documentation for comprehensive instructions o
...
 
 == Fixes
 
-* Fixed layout on Members Listing screen when slow operations are detected. [MC-2918]
+* Fixed layout on the Members Listing screen when slow operations are detected. [MC-2918]
 * Fixed links in license expiration popups. [MC-2719]
-* Jet job Received Items count no longer increases the stream has been cancelled. [MC-2761]
-* Entry cost is no longer shown with fraction of bytes. [MC-2084]
+* Jet job Received Items count no longer increases after the stream is cancelled. [MC-2761]
+* Entry cost is no longer shown as a fraction of bytes. [MC-2084]
```
Comment (promanenko) [inline docs/modules/release-notes/pages/5-5-0.adoc:41]: I think `as a` is misleading here

File: docs/modules/release-notes/pages/5-5-0.adoc [hunk 5]
```diff
@@ -21,26 +21,26 @@ a custom run script, refer to the documentation for comprehensive instructions o
...
 * Quick zoom in the chart no longer results in page scrolling and faded details. [MC-2486]
-* Turning off SQL Browser no longer returns user to Dashboard instead of the previous page. [MC-2678]
-* An error toast is shown now when user tries to add an expired license. [MC-2773]
-* Dropdowns no longer break WAN Replication layout when long values are selected. [MC-2784]
-* The number of User Code Namespaces displayed is no longer capped at 5. [MC-2892]
\ No newline at end of file
+* Turning off SQL Browser no longer returns the user to Dashboard instead of the previous page. [MC-2678]
+* An error toast is shown now when the user tries to add an expired license. [MC-2773]
```
Comment (promanenko) [inline docs/modules/release-notes/pages/5-5-0.adoc:44]: is there some particular user so we require `the` here? I understand possible `a` here

File: docs/modules/release-notes/pages/5-5-0.adoc [hunk 6]
```diff
@@ -21,26 +21,26 @@ a custom run script, refer to the documentation for comprehensive instructions o
...
-* Added a link to the License Feature CP Section if CP Subsystem is disabled. [MC-2847]
+* Added Slow Operations details to the Member Listing screen. [MC-2783]
+* Added a link to the License Feature CP Section if the CP Subsystem is disabled. [MC-2847]
 * Added an ability to a WAN Replication reference of an IMaps. [MC-2729]
 * Added an ability to configure `consistency-check-strategy` during dynamic creation of a WAN publisher. [MC-243]
-* Added an ability to load OIDC configuration params from an OIDC configuration endpoint. [MC-2733]
+* Added an ability to load OIDC configuration parameters from an OIDC configuration endpoint. [MC-2733]
 
-NOTE: in CLustered REST API `memberConnection` allowed field values have been changed from `ALL` | `<member-address>` to
+NOTE: in Clustered REST API `memberConnection` allowed field values have been changed from `ALL` | `<member-address>` to
```
Comment (promanenko) [inline docs/modules/release-notes/pages/5-5-0.adoc:33]: fixed

File: docs/modules/release-notes/pages/5-5-0.adoc [hunk 7]
```diff
@@ -21,26 +21,26 @@ a custom run script, refer to the documentation for comprehensive instructions o
...
 * HealthCheck icon now reflects the highest severity issue detected. [MC-2867]
 * Added cpDirectToLeader property value to Client screen. [MC-2902]
-* Added Slow Operations details to Member Listing screen. [MC-2783]
-* Added a link to the License Feature CP Section if CP Subsystem is disabled. [MC-2847]
+* Added Slow Operations details to the Member Listing screen. [MC-2783]
+* Added a link to the License Feature CP Section if the CP Subsystem is disabled. [MC-2847]
 * Added an ability to a WAN Replication reference of an IMaps. [MC-2729]
 * Added an ability to configure `consistency-check-strategy` during dynamic creation of a WAN publisher. [MC-243]
-* Added an ability to load OIDC configuration params from an OIDC configuration endpoint. [MC-2733]
+* Added an ability to load OIDC configuration parameters from an OIDC configuration endpoint. [MC-2733]
```
Comment (promanenko) [inline docs/modules/release-notes/pages/5-5-0.adoc:31]: fixed

File: docs/modules/release-notes/pages/5-5-0.adoc (hunk 2 above)
Comment (oliverhowell) [reply docs/modules/release-notes/pages/5-5-0.adoc:6]: I know it's a paste but I rewrote this in the main PR and left a couple of questions too

**Pull request closed without merging**